3rdparty/celt-0.7.0-build/CMakeLists.txt

```diff
@@ -68,3 +68,13 @@ target_sources(celt PRIVATE
 )
 
 target_disable_warnings(celt)
+
+# We have to explicitly link to the "m" (math) library.
+# See https://stackoverflow.com/q/1033898/3907364 for why
```

Comment:
Please note that your user ID is in the link.

Reply:
Yeah I don't mind. It only helps earning me badges there :D